string id
float64 x
float64 y
float64 weight
string drone_id
